Modules/TPC/run/tpcQCTracks_direct.json

```
@@ -35,7 +33,7 @@
           "query": "inputTracks:TPC/TRACKS/0"
         },
         "taskParameters": {
-          "myOwnKey": "myOwnValue"
+          "EtaCut": "1.",  "nClusterCut": "60", "dEdxTot": "20."
```

Perhaps a bit more verbose and consistent (either all with `Cut` or without)
```suggestion
          "AbsEtaCut": "1.",  "minNClusterCut": "60", "mindEdxTotCut": "20."
```
or
```suggestion
          "cutAbsEta": "1.",  "cutMinNCluster": "60", "cutMindEdxTott": "20."
```
or
```suggestion
          "AbsEta": "1.",  "minNCluster": "60", "mindEdxTot": "20."
```

---

yeah newer version is there in the commit with  cut as prefix